### PR TITLE
Enable remote-debugging-port parameter

### PR DESCRIPTION
**Pull request by aaron0x**
If this parameter is set, don't force it to be 0.

### CODE REVIEW

**Comment by aaron0x** [comment]
When user also sets remote-debugging-port, there are two remote-debugging-port passed to chrome, and chrome only take the later one, that's 0. debugging port can never be set.

File: allocate.go [hunk 1]
```diff
@@ -377,6 +379,11 @@ func UserAgent(userAgent string) ExecAllocatorOption {
 	return Flag("user-agent", userAgent)
 }
 
+// RemoteDebuggingPort is the command line option to set remote-debugging-port.
+func RemoteDebuggingPort(port string) ExecAllocatorOption {
```
Comment (mvdan) [inline allocate.go]: Remove this API. Most users shouldn't be setting the parameter at all, and those who know what they're doing can just call Flag directly.

Comment (aaron0x) [reply allocate.go]: ok

**Comment by mvdan** [comment]
Thanks. The builds are broken in master; that's unrelated to your change, which seems pretty safe and sensible to me.

**Pull request merged**